dix/devices.c

```diff
@@ -2446,31 +2450,25 @@ ProcGetMotionEvents(ClientPtr client)
                 coords[nEvents].x = coords[i].x - pWin->drawable.x;
                 coords[nEvents].y = coords[i].y - pWin->drawable.y;
                 nEvents++;
+
+                /* write xTimecoord */
+                x_rpcbuf_write_CARD32(&rpcbuf, coords[i].time);
+                x_rpcbuf_write_INT16(&rpcbuf, coords[i].x - pWin->drawable.x);
+                x_rpcbuf_write_INT16(&rpcbuf, coords[i].y - pWin->drawable.y);
```

Comment:
We probably don’t need to keep writing to `coords[nEvents]` (above) with this change.

Reply:
you're right. just forgot to drop these lines.